CppCoreGuidelines.md

```diff
@@ -21743,7 +21743,7 @@ Often, messy old code runs unnecessarily slowly because it requires outdated com
 In many cases, automated "modernizer"-style tool support would be required for major upgrade efforts.
 
 The purpose of modernizing code is to simplify adding new functionality, to ease maintenance, and to increase performance (throughput or latency), and to better utilize modern hardware.
-Making code "look pretty" or "follow modern style" are not by themselves reasons for change.
```

Comment:
We think "pretty" is clearer, it was specifically chosen to be a subjective term, whereas "better" is a stronger judgment that claims to be objective.

Reply:
reverted